.github/workflows/ci.yml

```diff
@@ -45,6 +45,11 @@ jobs:
             os: ubuntu-latest
             experimental: false
             nox-session: test_brotlipy
+          # Test CPython with a broken hostname_checks_common_name
+          - python-version: 3.9.2
+            os: ubuntu-latest
+            experimental: false
+            nox-session: test-3.8
```

Comment:
Should be test-3.9?

Reply:
Nice catch. With the fix, we do run the tests with 3.9.2. Wasn't noticed by coverage as Python 3.7 still covers this for now.